README.md

```diff
@@ -162,7 +162,7 @@ Publishes any kind of message encrypted to the group. The function wraps `ssb.db
 
 ### `ssb.tribes2.listMembers(groupId, { live }) => source`
 
-Returns a pull stream source listing every known member of the group with id `groupId`. Note: lists members whether or not they've accepted the invite. If `live` is true, then it keeps the stream open and also outputs new members.
+Returns a pull stream source listing the root feed id of every member of the group with id `groupId`. Note: lists members whether or not they've accepted the invite. If `live` is true, then it keeps the stream open and also outputs new members. If `live` is true and a member gets excluded, it outputs an object on the format `{ excluded: excludedPeerRootFeedId }`.
```

Comment:
I personally really dislike streams which have varied output like this.
in the live case you're forcing the person to handle a new type.

I think you should just leave the output in `{ added: [], excluded[] }` form, then the fields are always there, and instead of going `isString or isObject, then do this or that` the person can straight up just up `update.added` and trust it's always there.

Reply:
Also this might be pet peeve and I will not die if this is merged, I will just be annoyed whenever I see type checking

Reply:
why is it annoying? it feels like a footgun. You decide to change to "live" and your stream pipeline that was working now explodes

Reply:
ah i guess that's an option, to have it on object form. i think i wanted more backwards compat. but

1. do you think it should have this object form only on `live` or also when not live? because not live i think excluded would always be empty (if it gets filled you get a new epoch) although that brings me to my second question...
2. should the `added` list be a list of the members added just now or every time it's emitted it should be a complete list of all members? if it's always a complete list then we could just skip the `excluded` list and just return `Stream<AllCurrentMembers[]>`. Since the list is very unlikely to be >200 member ids long it wouldn't need to be a stream of a stream right? Although `Stream<AllCurrentMembers[]>` would just be a 1 item long stream when not live, feels weird?